alerter/alerter.cs

```diff
@@ -1,29 +1,45 @@
 ﻿using System;
+using System.Diagnostics;
 
-namespace AlerterSpace {
-    class Alerter {
+namespace AlerterSpace 
+{
+    class Alerter 
+    {
         static int alertFailureCount = 0;
-        static int networkAlertStub(float celcius) {
-            Console.WriteLine("ALERT: Temperature is {0} celcius", celcius);
+        static int networkAlertStub(float celcius) 
+        {            
             // Return 200 for ok
             // Return 500 for not-ok
-            // stub always succeeds and returns 200
-            return 200;
+            // stub always succeeds and returns 200  
+            if (celcius <= 200)
+            {
+                return 200;
+            }
+            else
+            {
+                Console.WriteLine("ALERT: Temperature is {0} celcius", celcius);
+                return 500;
+            }                 
         }
-        static void alertInCelcius(float farenheit) {
+        static void alertInCelcius(float farenheit) 
+        {
             float celcius = (farenheit - 32) * 5 / 9;
             int returnCode = networkAlertStub(celcius);
-            if (returnCode != 200) {
+            if (returnCode != 200) 
+            {
                 // non-ok response is not an error! Issues happen in life!
                 // let us keep a count of failures to report
                 // However, this code doesn't count failures!
                 // Add a test below to catch this bug. Alter the stub above, if needed.
                 alertFailureCount += 0;
             }
         }
-        static void Main(string[] args) {
+        static void Main(string[] args) 
+        {
             alertInCelcius(400.5f);
+            Debug.Assert(alertFailureCount == 1);
             alertInCelcius(303.6f);
+            Debug.Assert(alertFailureCount == 0);

```

Comment:
Alert Failure count resetting to zero is not possible, either the count remains same as before or increases by one.
Is your test code expecting cumulative failure count? or just a flag which is either 0 or 1?